client/src/components/EditableAttribute/EnumAttribute.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Select from 'react-select';
+
+const EnumAttribute = ({
+  value,
+  valueOptions,
+  attributeLabel,
+  isDisabled,
+  onChange,
+}) => {
+  const onValueChange = option => {
+    onChange(option, attributeLabel);
+  };
+
+  return (
+    <div>
+      <p>{attributeLabel}</p>
```

Comment:
We could def do this much later, but would it be possible to use the pretty-printed header names here? i.e., "First Name" instead of "firstName".

I know @amit-sawhney added the "pretty" header names in `Table.js`, so we should decide where we store those names so that both the table and the member profile page use one source of info for them.

Reply:
Yeah, we should definitely decide whether we're gonna store them on frontend or backend... maybe we should just plan to discuss this when we meet next week